crates/cheatnet/src/cheatcodes/mod.rs

```diff
@@ -67,6 +67,6 @@ impl From<EnhancedHintError> for CheatcodeError {
 
 #[derive(Debug, PartialEq, Clone)]
 pub struct ContractArtifacts {
-    pub sierra: String,
+    pub sierra: Option<String>,
```

Comment:
How class hash is calculated without sierra?

Reply:
Didn't know that, if we need both we can just get both I guess

Reply:
Okay, so it seems that actually `get_class_hash` is in fact `get_compiled_class_hash` and we use the `compiled_class_hash` instead of `class_hash` for contract identification  xD

Reply:
I'll try changing these so we calculate the correct values